### DISCUSSION TITLE
spacy models break conda environment deserialization

### DISCUSSION THREAD

**Discussion by DSLituiev** [explosion/spaCy#9616] [Help: Installation] [answered]
## How to reproduce the behaviour

1. Install a spacy model in a conda environment
2. Serialize the environment
`conda env export > environment.yml`
3. Deserialize the environment
 `conda env create -f environment.yml`

Can this be solved by publishing models to PYPI?

**Comment by polm** [comment]
We can't publish the models to PyPI because 1. they're too big 2. PyPI isn't interested in hosting data-only packages. So we'd have to address this in another way.

**Comment by adrianeboyd** [comment, marked as answer]
Because they're not on pypi, you have to specify the model dependencies differently for `pip`, see: https://spacy.io/usage/models#models-download

Or with the new wheel packages:

```
  - pip:
    - https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.1.0/en_core_web_sm-3.1.0-py3-none-any.whl
```
